warehouse/accounts/views.py

```diff
@@ -1462,8 +1462,7 @@ def add_pending_github_oidc_publisher(self):
         publisher_already_exists = (
             self.request.db.query(PendingGitHubPublisher)
             .filter_by(
-                repository_name=form.repository.data,
-                repository_owner=form.normalized_owner,
+                repository=form.repository.data,
```

Comment:
This isn't going to work, unfortunately -- `repository` isn't a field on a `PendingGitHubPublisher`.

Instead, we should do something similar to what was already in place: the form model should provide `normalized_owner` and `normalized_repository` attributes that get computed from `form.repository.data`.